diff -up KPackageKit/Common/KpkStrings.cpp.pk_api KPackageKit/Common/KpkStrings.cpp
--- KPackageKit/Common/KpkStrings.cpp.pk_api	2008-10-15 17:38:27.000000000 -0500
+++ KPackageKit/Common/KpkStrings.cpp	2008-10-21 10:09:07.000000000 -0500
@@ -549,8 +549,8 @@ QString KpkStrings::groups(Client::Group
 	    return i18n("Documentation");
         case Client::Electronics :
 	    return i18n("Electronics");
-        case Client::MetaPackages ://TODO check this one
-	    return i18n("Package collections");
+//        case Client::MetaPackages ://TODO check this one
+//	    return i18n("Package collections");
         case Client::Security :
 	    return i18n("Security");
         case Client::Servers :
@@ -606,8 +606,8 @@ KIcon KpkStrings::groupsIcon(Client::Gro
 	    return KIcon("applications-education-language");
         case Client::Maps :
 	    return KIcon("Maps");//FIXME
-        case Client::MetaPackages :
-	    return KIcon("unknown");//FIXME
+//        case Client::MetaPackages :
+//	    return KIcon("unknown");//FIXME
         case Client::Multimedia :
 	    return KIcon("applications-multimedia");
         case Client::Network :
