diff -up KPackageKit/Desktop/kpackagekit.desktop.kdeinit KPackageKit/Desktop/kpackagekit.desktop
--- KPackageKit/Desktop/kpackagekit.desktop.kdeinit	2008-10-15 17:38:27.000000000 -0500
+++ KPackageKit/Desktop/kpackagekit.desktop	2008-10-20 13:11:39.000000000 -0500
@@ -3,6 +3,7 @@ Type=Application
 Icon=applications-other
 Exec=kpackagekit -caption "%c" %i %F
 X-DBUS-StartupType=Unique
+X-DBUS-ServiceName=org.kde.KPackageKit
 
 Name=KPackageKit
 GenericName=Software Management
